lib/seek/extended_metadata_type/extended_metadata_type_schema.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "title": {
+      "type": "string"
+    },
+    "supported_type": {
+      "type": "string",
+      "enum": [
+        "Investigation",
+        "Study",
+        "Assay",
+        "Collection",
+        "DataFile",
+        "Document",
+        "Event",
+        "Model",
+        "Presentation",
+        "Sop",
+        "Project",
+        "ExtendedMetadata"
+      ]
+    },
+    "enabled": {
+      "type": "boolean"
+    },
+    "attributes": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "oneOf": [
+          {
+            "properties": {
+              "title": {
+                "type": "string"
+              },
+              "required": {
+                "type": "boolean"
+              },
+              "type": {
+                "type": "string",
+                "enum": [
+                  "Date time",
+                  "Date",
+                  "Real number",
+                  "Integer",
+                  "Web link",
+                  "Email address",
+                  "Text",
+                  "String",
+                  "ChEBI",
+                  "ECN",
+                  "MetaNetX chemical",
+                  "MetaNetX reaction",
+                  "MetaNetX compartment",
+                  "InChI",
+                  "ENA custom date",
+                  "Boolean",
+                  "URI",
+                  "DOI",
+                  "NCBI ID",
+                  "Registered Strain",
+                  "Registered Data file"
```

Comment:
these can't be validated through the schema, as it depends on how the server is configured and people may have added their own or removed some.